love/src/components/ESS/AuxTel/AuxTelESS.container.jsx

```diff
@@ -99,88 +100,157 @@ const getGroupNames = (salindexList, option) => {
   return [...new Set(subscriptions)];
 };
 
+/**
+ * 
+ * @param {dict} prevParse dict data of the sensors
```

Comment:
Please change dict per object
```suggestion
 * @param {object} prevParse dict data of the sensors
```
https://jsdoc.app/tags-param.html